ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main-periodics.yaml

```diff
@@ -1364,3 +1364,86 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build01
+  cron: 0 1 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: openshift-knative
+    repo: serverless-operator
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: ocp4.13-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.13"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-knative-serverless-operator-main-ocp4.13-lp-interop-operator-e2e-interop-aws-ocp413
+  reporter_config:
+    slack:
+      channel: '#serverless-qe'
```

Comment:
This line was previously different and pointed to a different Slack channel. Now it will be sending messages to serverless-qe channel. I previously asked who will be watching this job in https://github.com/openshift/release/pull/36697#issuecomment-1480690502
With this PR, does the workflow between PIT QE and Serverless QE change?
Can we stop using our former jenkins jobs and focus on this CI stuff and slack notifications?
If you're not able to answer this please give me some contact information to somebody who can.

I won't stamp this PR until my questions are answered.

Aside from that this PR looks alright.